### PR TITLE
Rover fix brushed for normal rover

### PR DESCRIPTION
**Pull request by khancyr**
This fix brushed for normal rover
add a bitmask to select which relay to use
fix SERVO_REVERSE with Brushed

### CODE REVIEW

**Comment by rmackay9** [comment]
I wonder if we really need the mapping for the relays?  It's a hardware thing so we could just tell users to connect relay1 to the throttle or throttle-left motor.  relay2 to the throttle-right motor.

File: APMrover2/AP_MotorsUGV.cpp [hunk 1]
```diff
@@ -126,12 +126,12 @@ void AP_MotorsUGV::output(bool armed, float dt)
 void AP_MotorsUGV::output_regular(bool armed, float steering, float throttle)
 {
     // always allow steering to move
-    SRV_Channels::set_output_scaled(SRV_Channel::k_steering, steering);
+    output_throttle(SRV_Channel::k_steering, steering);
```
Comment (rmackay9) [inline APMrover2/AP_MotorsUGV.cpp]: output_throttle?

Comment (khancyr) [inline APMrover2/AP_MotorsUGV.cpp]: we should rename this function. It is need otherwise the steering cannot use brushed and relay

**Comment by khancyr** [comment]
@rmackay9 Yep the bitmaks is a little difficulte ... maybe seperating it into two : one for throttle/steering, the  other for throttleLeft/throttleRight. 
The problem is you can use the relay for throttle  and throttleLeft at the same time as the value aren't the same. Or for now, we forbide the combined use of normal and steering rover.

**Comment by rmackay9** [comment]
@khancyr, how do you feel about this alternative fix for the brushed relay issue?
https://github.com/rmackay9/rmackay9-ardupilot/commit/8f3135448fbebe80f5d08ea9b4483f49ca2781ee

**Comment by rmackay9** [comment]
@khancyr, Thanks for the review.  i will add the fabfs on the throttle and push to master.

I don't think we can actually mix regular steering with skid-steering because the attitude controller output scaling is different.  We don't have specific check to stop people from doing that but we probably should add that.

**Comment by rmackay9** [comment]
Merged the brushed motor fix so closing.

**Pull request closed without merging**